src/services/initializer-service.ts

```diff
@@ -24,111 +18,70 @@ export default class InitializerService {
   private static debug = debugFactory(InitializerService)
 
   private readonly options: InitializerOptions
+  private readonly serviceTools: ServiceTools
```

Comment:
Make dependencies explicit and don't wrap them in another object

Reply:
service tool signature now removed and service objects are now explicit to initializer service